style.css

```diff
@@ -1,8 +1,9 @@
 @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700,800&display=swap');
 
 :root {
-  --embark-blue: #0fa0b4;
   --embark-red: #b40000;
+  --embark-blue: #0fa0b4;
+  
```

Comment:
This change doesn't seem to do anything - the value is still defined as the same color, except now it's in a different and we have an extra new line after it. I would recommend not changing this if it has no effect on the page.